scripts/prepublish.js

```diff
@@ -49,7 +49,8 @@ function validateFile(fileName) {
 
         res = `${sha512}  ${fileName}`;
     } else {
-        res = exec(`shasum -a 512 ${fileName}`).trim();
+        
+        res = exec(`command -v shasum >/dev/null 2>&1 && shasum -a 512 ${fileName} || sha512sum -a 512 ${fileName}`).trim();
```

Comment:
I don't think that `sha512sum` needs the `-a 512` parameter

Reply:
yes, it needn't parameter